subprojects/core-api/src/main/java/org/gradle/api/initialization/Settings.java

```diff
@@ -422,20 +422,20 @@ default void includeFlat(String... projectNames) {
 
 
     /**
-     * Returns the software type conventions object for this build.
+     * Returns the model defaults object for this build.
      *
      * @since 8.10
      */
     @Incubating
-    Conventions getConventions();
+    SharedModelDefaults getDefaults();
```

Comment:
This was introduced as part of the work to allow defaults to be applied in non-declarative files.  We only exposed this because there weren't good options for exposing this for Kotlin scripts in a non-public way.  It's not being announced, documented or promoted in any way at this point.  I would not view this as any indication that this stuff is still anything more than an experiment at this point.

Reply:
Do you think we could express something similar in the Javadoc for these methods?